Modules/socketmodule.c

```diff
@@ -6117,11 +6114,8 @@ PyDoc_STRVAR(ntohs_doc,
 "ntohs(integer) -> integer\n\
 \n\
 Convert a 16-bit unsigned integer from network to host byte order.\n\
-Note that in case the received integer does not fit in 16-bit unsigned\n\
-integer, but does fit in a positive C int, it is silently truncated to\n\
-16-bit unsigned integer.\n\
-However, this silent truncation feature is deprecated, and will raise an\n\
-exception in future versions of Python.");
+An OverflowError is raised if the received integer does not fit in\n\
```

Comment:
OverflowError is usually not mentioned specially in docstrings. See for example docstrings for ntohl/htonl.